### PR TITLE
chore: add more span attributes to screenshots

### PR DESCRIPTION
**Pull request by joaoviana**
Closes: N/A

### Description:

Add more span attributes to screenshot logger

### Reviewer actions

- [ ] I have manually tested the changes in the preview environment
- [ ] I have reviewed the code
- [ ] I understand that "request changes" will block this PR from merging

### CODE REVIEW

File: packages/backend/src/services/UnfurlService/UnfurlService.ts [hunk 1]
```diff
@@ -402,14 +420,14 @@ export class UnfurlService {
                             width: gridWidth ?? viewport.width,
                         });
                     }
-                    await page.on('requestfailed', (request) => {
+                    page.on('requestfailed', (request) => {
```
Comment (rephus) [inline packages/backend/src/services/UnfurlService/UnfurlService.ts:423]: why did you remove the await ?  in case of error, will it throw an `uncaught error` that can break the server ?

**Review by rephus — APPROVED** [review]

File: packages/backend/src/services/UnfurlService/UnfurlService.ts (hunk 1 above)
Comment (joaoviana) [reply packages/backend/src/services/UnfurlService/UnfurlService.ts:423]: Because the `await` had no effect on this expression, 

image

The error would be caught anyway by the `try/catch` wrap I believe
